README.md

```diff
@@ -10,122 +10,274 @@
+ .aggregate(averagingDouble(reading -> reading.temperature()))
+ .writeTo(Sinks.logger());
 
-* [Java](https://github.com/hazelcast/hazelcast)
-* [.NET](https://github.com/hazelcast/hazelcast-csharp-client)
-* [Python](https://github.com/hazelcast/hazelcast-python-client)
-* [C++](https://github.com/hazelcast/hazelcast-cpp-client)
-* [Node.js](https://github.com/hazelcast/hazelcast-nodejs-client)
-* [Go](https://github.com/hazelcast/hazelcast-go-client)
+hz.getJet().newJob(p).join();
+```
 
-Hazelcast also has first-class support for running on different
-cloud providers such as [AWS](https://github.com/hazelcast/hazelcast-aws),
-[GCP](https://github.com/hazelcast/hazelcast-gcp) 
-and [Azure](https://github.com/hazelcast/hazelcast-azure)
-as well as on [Kubernetes](https://github.com/hazelcast/hazelcast-kubernetes).
+Use the following command to deploy the application to the server:
 
-## Download
+```bash
+bin/hazelcast submit analyze-sensors.jar
+```
 
-You can download Hazelcast from
-[hazelcast.org](http://hazelcast.org/download/). Once you have
-downloaded, you can start the Hazelcast instance using the script
-`bin/start.sh`.
+Jet supports advanced streaming features such as [exactly-once processing](https://docs.hazelcast.com/hazelcast/latest/fault-tolerance/fault-tolerance.html) and
+[watermarks](https://docs.hazelcast.com/hazelcast/latest/architecture/event-time-processing.html).
+
+#### Data Processing using SQL
+
+Jet also powers the [SQL engine](link to SQL) in Hazelcast which can execute
+both streaming and batch queries. Internally, all SQL queries are converted to
+Jet jobs.
+
+```sql
+CREATE MAPPING trades (
+    id BIGINT,
+    ticker VARCHAR,
+    price DECIMAL,
+    amount BIGINT)
+TYPE Kafka
+OPTIONS (
+    'valueFormat' = 'json',
+    'bootstrap.servers' = 'kafka:9092'
+);
+SELECT ticker, ROUND(price * 100) AS price_cents, amount
+  FROM trades
+  WHERE price * amount > 100;
++------------+----------------------+-------------------+
+|ticker      |           price_cents|             amount|
++------------+----------------------+-------------------+
+|EFGH        |                  1400|                 20|
+```
 
-## Get Started
+### Messaging
 
-Hazelcast allows you to interact with a cluster using a simple API, for
-example you can use the Hazelcast Java Client to connect to a running
-cluster and perform operations on it:
+Hazelcast provides lightweight options for adding messaging to your application.
+The two main constructs for messaging are topics and queues.
 
-```java
-HazelcastInstance hz = HazelcastClient.newHazelcastClient();
-IMap<String, String> map = hz.getMap("my-distributed-map");
-map.put("key", "value");
-String current = map.get("key");
-map.putIfAbsent("somekey", "somevalue");
-map.replace("key", "value", "newvalue");
+#### Topics
+
+Topics provide a publish-subscribe pattern where each message is fanned out to
+multiple subscribers. 
+
+```python
```

Comment:
I would really like examples for Java as well just like you did above. I understand why you did it but it feels wrong to leave out Java completely. It's a small code snippet and maybe having two languages next to each other actually emphasizes the polyglot aspect even more.